src/style/components/_button.scss

```diff
@@ -1,20 +1,22 @@
 .cs-button {
     background-color: $button-bg-color;
     border: 0;
-    border-radius: 2px;
+    border-radius: .125rem;
     color: $button-text-color;
     cursor: pointer;
     display: inline-block;
-    font-size: 14px;
+    font-size: .875rem;
     font-weight: bolder;
-    height: 36px;
-    line-height: 36px;
-    padding: 0 26px;
+    height: 2.25rem;
+    line-height: 2.25rem;
+    padding: 0 1.625rem;
```

Comment:
A suggestion here: What about first putting font-size with REMs (As you did) and then putting the rest of the sizes in EMs? This way we're taking a "base" size from root using REM and then applying the rest of the sizes based on this size.